kolibri/plugins/learn/assets/src/actions.js

```diff
@@ -36,7 +36,35 @@ const fetchNodes = ({ dispatch }, id) => {
   });
 };
 
+
+const searchNodes = ({ dispatch }, params, page) => {
+  // Get the collection from ContentNodeResource.
+  const contentCollection = Kolibri.resources.ContentNodeResource.getCollection();
+  const wantedFields = ['pk', 'title', 'kind', 'instance_id', 'content_id', 'description', 'files'];
```

Comment:
OK, so we might as well just return all the fields then, to save a second request, I would say. That way we have all the data client side after the search result.